processAndUpload.ts

```diff
@@ -105,162 +101,211 @@ function shapeData(node: Node) {
   };
 }
 
-
-
-export async function processCleanedLogs(transactions: Transactions, cleanedLogs: APLogs[]) {
+export async function processCleanedLogs(
+  transactions: Transactions,
+  cleanedLogs: APLogs[]
+) {
   // console.log('processCleanedLogs function called');
-  //   console.log('Processing cleaned logs...'); 
-    // console.log('transactions.edges:', transactions.edges);
-    console.log('cleanedLogs:', `${JSON.stringify(cleanedLogs,replacer,2)}`);
-    for (const edge of transactions.edges) {
-        for (const log of cleanedLogs) {
-          console.log('log:', log);
-            if (!log.args) {
-                console.log(`Skipping log due to missing args: ${JSON.stringify(log)}`);
-                continue;
+  //   console.log('Processing cleaned logs...');
+  // console.log('transactions.edges:', transactions.edges);
+  console.log('cleanedLogs:', `${JSON.stringify(cleanedLogs, replacer, 2)}`);
+  for (const edge of transactions.edges) {
```

Comment:
the reason this nested loop structure exists is because i wanted to put the arweave ids in each Event Table. we have to get these ids from that transaction query that populates the Transactions prisma table. however, doing so slows down the script significantly. so i have two questions: 

1. do we need arweave id's stored in the Event Tables
2. is there a way to optimize those loops?

Reply:
@0xTranqui @salieflewis

Reply:
we're removing top level for loop. arweave id's do not need to be in event tables.